device/Microsoft.Azure.Devices.Client.Shared/IoTHubClientDiagnostic.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace Microsoft.Azure.Devices.Client
+{
+    using System;
+    using System.Globalization;
+    using Microsoft.Azure.Amqp;
+
+    class IoTHubClientDiagnostic
```

Comment:
Please explicitly add public/internal/private to all members.

Reply:
Thanks for your suggestions, I will fix it